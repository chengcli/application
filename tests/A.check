       3      22     192 A.out
