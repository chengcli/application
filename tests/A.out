Log, XXX, A, 1., "First step", 0
Log, XXX, A, 2., "Second step", 0
